src/Files.Uwp/ViewModels/ColumnsViewModel.cs

```diff
@@ -269,5 +299,31 @@ public void TryMultiplySize(double factor)
 
             UserLength = new GridLength(setLength);
         }
+
+        public override bool Equals(object obj)
+        {
+            if (obj == null)
+            {
+                return false;
+            }
+            if (obj == this)
+            {
+                return true;
+            }
+            if (obj is ColumnViewModel model)
+            {
+                return (
+                    model.UserCollapsed == this.UserCollapsed &&
+                    model.Length.Value == this.Length.Value &&
+                    model.LengthIncludingGridSplitter.Value == this.LengthIncludingGridSplitter.Value &&
+                    model.UserLength.Value == this.UserLength.Value);
+            }
+            return base.Equals(obj);
+        }
+
+        public override int GetHashCode()
+        {
+            return base.GetHashCode();
+        }
```

Comment:
As above, is this needed and if so, equals-hashcode contract might be a problem.

Reply:
Keeping comment below.